server/config.yaml

```diff
@@ -3,5 +3,5 @@ database:
   port: 5432
   host: "localhost"
   user: "postgres"
-  password: "summer2020"
+  password: "1234"
```

Comment:
Rather than changing the config file, you should update your database to use what's in the config file. Something like a config file should be kept static as much as possible. Otherwise, the next person to come along will change it to match their own setting.

More ideally, we would never put passwords in a config file (for security reasons) but we have it here the purpose of quick development.

Reply:
to add onto what joseph said, the config file is just default values for a developer to use to get the app up and running. You don't have to use the same user/password for your postgres database but you also shouldn't commit any changes that reflect your personal dev environment, if it's different than the default.

Reply:
So, do I change the password back to "summer2020" when I push it? Or what would I do?

Reply:
Yup, change it back to `summer2020` now and then update your postgres user to use that password.

https://www.postgresql.org/docs/8.0/sql-alteruser.html